addons/point_of_sale/static/src/xml/Popups/MoneyDetailsPopup.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+    <t t-name="MoneyDetailsPopup" owl="1">
+        <div class="popup money-details invisible">
+            <main class="body">
+                <div class="money-details-title">
+                    Coins/Bills
+                </div>
+                <div class="money-details-info">
+                    <div t-foreach="[firstHalfMoneyDetails, lastHalfMoneyDetails]" t-as="moneyDetailsList">
```

Comment:
You need t-key. I suggest the following if it works.
```suggestion
                    <div t-foreach="[{key: 'first', val: firstHalfMoneyDetails}, {key: 'second', val: lastHalfMoneyDetails}]" t-as="moneyDetailsList" t-key="moneyDetailsList.key">
                        <t t-foreach="moneyDetailsList.val" t-as="moneyValue" t-key="moneyValue">
```

Reply:
Apparently I don't need to, I don't get the warning thanks to the `t-key` below

Reply:
Okay.